.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: php
 sudo: yes
-dist: trusty
+dist: bionic
 
 env:
-  - PHPVER="7.1.27"
+  - PHPVER="7.3.8"
```

Comment:
Should we be testing against both PHP 7.1 and PHP 7.3? Or the support for the former is being dropped?

Reply:
There are incompatible changes, e.g. in signatures - `php_register_url_stream_wrapper_volatile`. I suppose we'll need different releases for different PHP versions.